docs/fx-core/switch-tenant-or-subscription-help.md

```diff
@@ -4,7 +4,7 @@ This doc is to help you understand what will happen when provisioning in an alre
 
 ## Switch Microsoft 365 Account
 ### Local Debug
-You could run local debugging for a Teams project with one M365 account and then easily switch to another account for further local debugging. To do this, you only need to:
+You could run local debugging for a Teams project with one M365 tenant and then easily switch to another tenant for further local debugging. To do this, you only need to:
```

Comment:
Do not use words M365 but use Microsoft 365, I need to create a bug for this. recently got feedback from M365 developer program PM that we should never use M365 anywhere.

Reply:
Fixed.